FileSystem/FileIOTests.js

```diff
@@ -0,0 +1,113 @@
+function file_io_run_all_tests() {
+  QUnit.module("FileIO");
+
+  file_open_close_tests();
+//  file_print_tests();
```

Comment:
remove or uncomment

Reply:
Actually this is used to enable/disable the tests. Required to speed up testing as Appscript is slow

Reply:
Fixed